pkgs/tools/audio/pw-volume/default.nix

```diff
@@ -0,0 +1,33 @@
+{ lib
+, fetchFromGitHub
+, fetchurl
+, rustPlatform
+}:
+
+rustPlatform.buildRustPackage rec {
+  pname = "pw-volume";
+  version = "0.4.0";
+
+  src = fetchFromGitHub {
+    owner = "smasher164";
+    repo = pname;
+    rev = "v${version}";
+    sha256 = "sha256-u7Ct9Kfwld/h3b6hUZdfHNuDGE4NA3MwrmgUj4g64lw=";
+  };
+
+  cargoPatches = [ (fetchurl {
```

Comment:
```suggestion
  cargoPatches = [ 
    (fetchurl {
```

Reply:
force pushing a nixpkgs-fmt, well see if it succeeds